troposphere/static/js/components/Master.react.js

```diff
@@ -1,142 +1,140 @@
-define(function (require) {
-  "use strict";
-
-  var React = require('react/addons'),
-      stores = require('stores'),
-      Backbone = require('backbone'),
-      context = require('context'),
-      globals = require('globals'),
-      Header = require('./Header.react'),
-      Footer = require('./Footer.react'),
-      actions = require('actions'),
-      showUnsupportedModal = require('modals/unsupported/showUnsupportedModal.js'),
-      modernizrTest = require('components/modals/unsupported/modernizrTest.js'),
-      NullProject = require('models/NullProject');
-
-  // Routing
-  var Router = require('react-router'),
-    RouteHandler = Router.RouteHandler;
-
-  return React.createClass({
-    displayName: "Master",
+import React from 'react/addons';
```

Comment:
It'd be helpful for the merge into `master` if we change this `import` to just `'react'` and not `'react/addons'`